components/tooltip/Tooltip.js

```diff
@@ -230,11 +198,6 @@ const tooltipFactory = (options = {}) => {
           ...other
```

Comment:
**PP** Would really prefer using `lodash.omit` here instead of assigning useless variables.

Reply:
Done!

 :ok: by @driesd on 27/11/2017 14:24:08